plugins/BEdita/Core/src/Model/Behavior/CustomPropertiesBehavior.php

```diff
@@ -212,15 +212,12 @@ protected function isFieldSet($entity, $field)
     {
         $allProperties = $entity;
         if ($entity instanceof EntityInterface) {
-            $hidden = $entity->getHidden();
-            try {
-                $entity->setHidden([]);
-                $allProperties = $entity->toArray();
-            } finally {
-                $entity->setHidden($hidden);
-            }
+            $allProperties = array_flip(array_merge(
```

Comment:
But in this way we take all entity properties not just those set. Is it correct?